tests/semanticdb/metac.expect

```diff
@@ -935,7 +935,7 @@ _empty_/Enums.Suits.Clubs. => case val static enum method Clubs
 _empty_/Enums.Suits.Diamonds. => case val static enum method Diamonds
 _empty_/Enums.Suits.Hearts. => case val static enum method Hearts
 _empty_/Enums.Suits.Spades. => case val static enum method Spades
-_empty_/Enums.Suits.derived$CanEqual. => implicit lazy val method derived$CanEqual
+_empty_/Enums.Suits.derived$CanEqual. => lazy val given method derived$CanEqual
```

Comment:
Is this really given? Can they be lazy?
```suggestion
_empty_/Enums.Suits.derived$CanEqual. => lazy val given method derived$CanEqual
```

Reply:
Yes, this is actually `Given`. This symbol is automatically generated `given` value by `drives CanEqual`

```scala
enum Suits derives CanEqual:
  case Hearts, Spades, Clubs, Diamonds

// `drives CanEuqual` will automatically generate given instance something like
lazy given val derived$CanEqual: CanEqual[Enums.Suits, Enums.Suits] = 
   CanEqual.derived
```

https://dotty.epfl.ch/docs/reference/contextual/derivation.html

Reply:
Yes, parameterless givens are mapped to lazyvals by default. Some of them are mapped to defs when we can prove that that does not change semantics, so that we can omit generating a field.